### ISSUE TITLE
Sending images fails

### ISSUE THREAD

**Issue by timsueberkrueb** [tim-sueberkrueb/ubuntu-hangups#36]
```
Task exception was never retrieved
future: <Task finished coro=<send_message() done, defined at /opt/click.ubuntu.com/ubuntu-hangups.timsueberkrueb/0.3/lib/py/hangups/conversation.py:222> exception=KeyError('sessionStatus',)>
Traceback (most recent call last):
  File "/usr/lib/python3.4/asyncio/tasks.py", line 238, in _step
    result = next(coro)
  File "/opt/click.ubuntu.com/ubuntu-hangups.timsueberkrueb/0.3/lib/py/hangups/conversation.py", line 244, in send_message
    image_id = yield from self._client.upload_image(image_file)
  File "/opt/click.ubuntu.com/ubuntu-hangups.timsueberkrueb/0.3/lib/py/hangups/client.py", line 243, in upload_image
    return (json.loads(res2.body.decode())['sessionStatus']
KeyError: 'sessionStatus'
```

**Comment by timsueberkrueb** [comment]
Can't reproduce this anymore ... Not sure whether this occurs irregular or is fixed now.

**Comment by StuntsPT** [comment]
This was fixed in upstream [hangups  #182](https://github.com/tdryer/hangups/pull/182). It was using http to send the images, but the hangouts protocol now only accepts https.
I'd say you can't reproduce since in another commit you mentioned updating hangups. =-)

**Comment by timsueberkrueb** [comment]
@StuntsPT ah, okay, thanks!
